### ISSUE TITLE
Two instances of CodeLite opened up, when run for first time.

### ISSUE THREAD

**Issue by green-eyed-bear** [eranif/codelite#1945]
When I ran CodeLite 12.0 for the first time, two instances of it opened up.  One was
on top of the other, so it was "hidden from view".

--------------------------
CodeLite Version:  12.0

OS:  Debian 9

Installation Method:
  1) Cloned repository.
  2) Set branch to "12.0".
  3) Manual build via "make/make install".

**Comment by eranif** [comment]
I am unable to reproduce this... I have deleted the `~/.codelite` folder to make it a clean install only once instance was launched.
Any tips on how to reproduce? If you delete the folder `~/.codelite` (or rename it) and start CodeLite again, can you reproduce this?

**Comment by green-eyed-bear** [comment]
Yes, I get 100% repro with this bug.  I tried again as you requested, and here's some more details.

Note:  When "setup wizard" runs, if I hit "cancel", the error does NOT show up.

These are the choices I selected from "setup wizard".
1) Development Profile: C/C++ development
2) Scanned for compilers:  Found GCC, clang, and Cross GCC
3) No change to custom colors.
4) No change to whitespace/indentation.
5) When I hit "finish", is when the 2nd window pops up over the first.
